core-web/libs/portlets/dot-content-drive/portlet/src/lib/store/dot-content-drive.store.ts

```diff
@@ -127,6 +128,12 @@ export const DotContentDriveStore = signalStore(
             getFilterValue(filter: string) {
                 return store.filters()[filter];
             },
+            setSelectedItems(items: DotContentDriveItem[]) {
+                patchState(store, { selectedItems: items });
+            },
+            updateSelectedItems(items: DotContentDriveItem[]) {
```

Comment:
How does this behave when you unselect everything?

Reply:
I'll called `setSelectItems([])`

Reply:
Do we still need the updateSelectedItems?